### PR TITLE
Fix typo in function sqrt

### PR DESCRIPTION
**Pull request by rushikeshacharya**
### What I did
Fix typo in docs/built-in-function.rst/ : sqrt
### Cute Animal Picture

Put a link to a cute animal picture inside the parenthesis-->

### CODE REVIEW

File: docs/built-in-functions.rst [hunk 1]
```diff
@@ -186,10 +186,10 @@ Returns ``sha256`` (SHA2 256bit output) hash of input.
     :param a:
     :type a: decimal, larger than 0.0
 
-    :output sqrt: decimal
+    :output decimal: decimal
```
Comment (fubuloubu) [inline docs/built-in-functions.rst]:
```suggestion
    :output b: decimal
```

Comment (rushikeshacharya) [reply docs/built-in-functions.rst]: Yeah. I missed this in the last commit.

Comment (fubuloubu) [reply docs/built-in-functions.rst]: You can commit this suggestion using the GitHub interface above if you want

**Comment by rushikeshacharya** [comment]
Please check the PR #1437

**Pull request closed without merging**